main.py

```diff
@@ -1,16 +1,27 @@
-# This is a sample Python script.
+from selenium import webdriver, common
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
 
-# Press ⌃R to execute it or replace it with your code.
-# Press Double ⇧ to search everywhere for classes, files, tool windows, actions, and settings.
+# Немного настроим наш браузер
```

Comment:
Please, use English-only comments here and below

Reply:
Fixed